### PR TITLE
Added Landing Page

### PR DESCRIPTION
**Pull request by dubeyaditya29**
Added Landing page for doc site.

https://user-images.githubusercontent.com/48158220/154768991-e1bec9e7-863c-4fcb-8e08-9eea4a5c3b98.mp4

### CODE REVIEW

File: Stylesheet/variable.css [hunk 1]
```diff
@@ -42,7 +42,117 @@ body {
...
+  display: flex;
+  gap: var(--size-3);
+}
+.title_space {
+  color: var(--color-royalBlue);
+}
+
+.sub_container-right--image {
+  width: 300px;
+  height: 300px;
```
Comment (Shivanshsharma01) [inline Stylesheet/variable.css]: can use rem here

File: index.html [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="./Stylesheet/final.style.css" />
+  </head>
+  <body>
+    <nav class="navbar">
+      <div class="logo">
```
Comment (Shivanshsharma01) [inline index.html:13]: It's good to wrap the main nav in a "header" tag.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+            </button>
+          </div>
+        </div>
+        <div class="installation_guide">
+          <h1 class="heading_main">Installation</h1>
+          <iframe
+            src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=seti&wt=none&l=auto&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%2520%253Cli%2520XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX%253C%252Fli%253E"
+            style="
+              width: 766px;
+              height: 205px;
```
Comment (Shivanshsharma01) [inline index.html]: can use rem here too

**Review by Shivanshsharma01 — APPROVED** [review]
The code is clean and well structured with good use of variables. Can implement suggested changes

File: index.html [hunk 4]
```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
```
Comment (piyushsh25) [inline index.html]: you can change the title to something meaningful !

**Review by piyushsh25 — APPROVED** [review]
Apart from the suggestion. Easy to read and clean code. You can merge.

File: Stylesheet/variable.css (hunk 1 above)
Comment (dubeyaditya29) [reply Stylesheet/variable.css]: yeah forgot to change it was trying out something thanks for pointing out. :)

File: index.html (hunk 4 above)
Comment (dubeyaditya29) [reply index.html]: Thanks piyush for pointing out will change the same.

**Pull request merged**